Workiva/vs-code-format-on-save
# Use dart executable instead of pub

## Motivation

We just adopt Dart 2.18 and with that, `pub` is no longer an executable. This PR switches it to `dart` so that consumers don't need a `pub` alias.

## Changes

- Switch from `pub` executable to `dart`
- Rebuild the extension as v1.0.2

## QA'ing
- [ ] Install the package and verify it works with Dart 2.18
    - Clone this repo
    - In the repo, run `code --install-extension over-react-format-on-save-1.0.2.vsix`
    - Your VS Code OverReach Format on Save extension should be v1.0.2 now. 
        - If it's not, you can run remove the v1.0.1 (or whatever version) from the extensions directory (`~/.vscode/extensions`), and it your IDE should use v1.0.2
    - Open a project in VS Code, being sure you're using Dart 2.18
    - Verify formatting works as expected
